Publish/Portal/PortalContentPost.py

```diff
@@ -566,17 +566,21 @@ def publish_user_items(portal, username, usercontentpath, old_hostname, new_host
         
         # Add/Update item
         if do_load_item:
-            item, old_source_id = load_item(portal, os.path.join(usercontentpath,"items", item_dir), overwrite_id)
-            newitems.append(item)
-            old_source_ids.append(old_source_id)         
+		
+			try : 
+				item, old_source_id = load_item(portal, os.path.join(usercontentpath,"items", item_dir), overwrite_id)
+				newitems.append(item)
+				old_source_ids.append(old_source_id)         
 
-            # Reassign item to target owner and folder
-            if os.path.exists(os.path.join(usercontentpath, "folders.json")):
-                os.chdir(usercontentpath)
-                foldersinfo = json.load(open('folders.json'))
-                foldername = publish_get_folder_name_for_item(item, foldersinfo)
-            
-            portal.reassign_item(item['id'], username, foldername)
+				# Reassign item to target owner and folder
+				if os.path.exists(os.path.join(usercontentpath, "folders.json")):
+					os.chdir(usercontentpath)
+					foldersinfo = json.load(open('folders.json'))
+					foldername = publish_get_folder_name_for_item(item, foldersinfo)
+				
+				portal.reassign_item(item['id'], username, foldername)
+			except Exception as err : 
+				print('--> IMPORTANT: Exception on item: ' + str(item))
```

Comment:
Thanks - note: you can also make a comment on a commit or the main [PR conversation page](https://github.com/Esri/ops-server-config/pull/732) if it doesn't make sense to comment on a line number directly